src/plugins/intel_npu/src/plugin/npuw/llm_infer_request.cpp

```diff
@@ -124,7 +124,8 @@ ov::npuw::LLMInferRequest::LLMInferRequest(const std::shared_ptr<ov::npuw::LLMCo
 }
 
 void ov::npuw::LLMInferRequest::prepare_for_new_conversation() {
-    fill_tensor<int64_t>(m_prefill_request->get_tensor(m_prefill_in_ports.at("input_ids")), 0);
+    const auto pad_token_id = m_npuw_llm_compiled_model->m_cfg.get<::intel_npu::NPUW_LLM_PAD_TOKEN_ID>();
+    fill_tensor<int64_t>(m_prefill_request->get_tensor(m_prefill_in_ports.at("input_ids")), pad_token_id);
```

Comment:
does it really matter what is pad token value if it's turned off via attention_mask ?

Reply:
Hello! Thanks a lot for the review, Anatolii has also confirmed that pad token id shouldn't influence the operation of model.